spec/interfaceDuckSpec.js

```diff
@@ -0,0 +1,43 @@
+//isItTheDuckingSame(executedFunction, expectedOutput, description)
+//isItTheDuckingSameArray(executedFunction, [expectedOutput], description)
+console.log("THESE ARE THE interface.js TESTS!")
+
+console.log("#displayCries")
+  //arrange
+  let testDiv = document.createElement("div");
+  testDiv.setAttribute('id', 'cryer-div');
+  document.body.appendChild(testDiv)
+
+  let stubbedData = [
+    { body: "This is a cry!", updated_at: "2021-02-16T19:48:17.065Z", user: {id: 34, handle: "kay"}},
+    { body: "This is another cry!", updated_at: "2021-02-16T19:48:17.065Z", user: {id: 34, handle: "kay"}}
+  ]
+  let testInterface = new Interface((url, callback) => {
+      callback(stubbedData)
```

Comment:
You found an approach to inject mock data into your interface that works but the reason you had to do that in the first place is that your `Interface` also takes care of displaying things instead of only doing API-related stuff. If you manage to move all of the display related code into `app.js` with some of the approaches I suggested in another comment, you won't have to inject mock data here.

You would still have to mock your calls to `fetch` which you could do manually without the use of a mocking library using an approach like the ones in this pill: https://github.com/makersacademy/course/blob/master/pills/mocking_in_javascript.md
Basically in your tests, you would overwrite `window.fetch` with a different function in your tests.

Other JS testing libraries like Jest have some helpers to do that a bit more easily but that's the general idea.